physionet-django/annotation/tests.py

```diff
@@ -1,3 +1,366 @@
-from django.test import TestCase
+import requests
+import json
+import os
+from django.utils import timezone
+from datetime import timedelta
 
-# Create your tests here.
+from project.models import PublishedProject, AccessPolicy, ProjectType, CoreProject
+from annotation.models import AnnotationCollection, AnnotationType
+from annotation.views import AnnotationCreateAPIView
+from annotation.views import (
+    AnnotationCollectionCreateAPIView,
+    AnnotationTypeCreateAPIView,
+)
+from user.models import User
+
+from django.test import RequestFactory, TestCase
+from rest_framework.test import APIRequestFactory, force_authenticate, APIClient
+from oauth2_provider.models import get_access_token_model, get_application_model
+from oauth2_provider.settings import oauth2_settings
+
+Application = get_application_model()
+AccessToken = get_access_token_model()
+
+CLEARTEXT_SECRET = "1234567890abcdefghijklmnopqrstuvwxyz"
+BASE_URL = "http://127.0.0.1:8000"
```

Comment:
This should be removed - the URL isn't necessarily correct. Instead use `from django.urls import reverse` (see other tests for examples)